jekyll/rails-addon.markdown

```diff
@@ -0,0 +1,152 @@
+---
+layout: default
+title: Addon for Rails
```

Comment:
We need to make an effort to standardize the naming to `Rails addon`. Somehow the name "Rails LSP" already spread, so let's ensure consistency.

```suggestion
title: Rails addon
```